frontend/packages/noobaa-storage-plugin/src/components/data-consumption-card/data-consumption-card.tsx

```diff
@@ -84,22 +86,28 @@ const DataConsumptionCard: React.FC<DashboardItemProps> = ({
   let maxVal: number;
   let chartData = [];
   let legendData = [];
+  let suffixLabel = '';
   const result = _.get(dataConsumptionQueryResult, 'data.result', []);
   if (result.length) {
     let maxData: BarChartData | any = {
       x: '',
       y: 0,
       name: '',
+      yOriginal: 0,
+      unit: '',
     };
     chartData = metricsChartDataMap[metricType][sortByKpi](result);
     legendData = metricsChartLegendDataMap[metricType][sortByKpi](chartData);
-    maxData = _.maxBy(chartData.map((data) => _.maxBy(data, 'y')), 'y');
-    maxVal = maxData.y;
-    maxUnit =
-      (sortByKpi === BY_IOPS || sortByKpi === BY_PHYSICAL_VS_LOGICAL_USAGE) &&
-      chartData.length === 2
-        ? humanizeDecimalBytes(maxVal).unit
-        : humanizeBinaryBytesWithoutB(maxVal).unit;
+    maxData = _.maxBy(chartData.map((data) => _.maxBy(data, 'yOriginal')), 'yOriginal');
+    const maxDataV = Number(maxData.yOriginal);
+    const maxDataVStr = maxDataV.toFixed(1);
+    let maxDataH = humanizeBinaryBytes(maxDataVStr);
+    if (sortByKpi === BY_IOPS) {
+      suffixLabel = numberInWords(maxDataV);
+      maxDataH = humanizeDecimalBytesWithoutB(maxDataVStr);
```

Comment:
@aruniiird I think we don't require `humanizeDecimalBytesWithoutB`. We can use `humanizeDecimalBytes` itself. All the dashboard are going to remove withoutB functionality

Reply:
But @cloudbehl , in the mock data (only for `I/O Operations`) we have only one character units... If you prefer not to edit 'units.js' file , I can revert the changes from 'units.js' and add the logic (to show one letter unit) into our card's code.

Snapshot from mock data

![image](https://user-images.githubusercontent.com/895234/62748405-1ef83100-ba76-11e9-8a67-7fa683dc922d.png)

Reply:
@cloudbehl I agree, the values should be closer to their label (I'm not very familiar with what styling options are available for the React component).

Also @cloudbehl PR #2296 which changes to `humanizeDecimalBytes` could use some help if you're able to take a look. 🙂 We definitely don't want to have a mix of different unit labels.

Reply:
can we use `humanizeDecimalBytes` here ?

Reply:
Acknowledged...
PS: will be removing all the changes from units.js file

Some screen shots...

![image](https://user-images.githubusercontent.com/895234/63690199-a214d780-c829-11e9-8488-74e3e74730be.png)

![image](https://user-images.githubusercontent.com/895234/63690237-beb10f80-c829-11e9-8739-fa5429248c7e.png)